src/components/controls/selects/SearchSelect/SearchSelect.stories.tsx

```diff
@@ -46,13 +31,57 @@ Basic.args = {
   label: "Label",
   options: basicOptions,
 }
+Basic.play = async ({ canvasElement }) => {
+  const canvas = within(canvasElement)
+  const label = canvas.getByText("Label")
+  const inputField = canvas.getByRole("textbox")
+  await expect(label).toBeTruthy()
+  await expect(inputField).toBeTruthy()
+}
+
+export const LargeDataset = Template.bind({})
+LargeDataset.args = {
+  label: "Label",
+  options: largeDataset,
+}
+LargeDataset.play = async ({ canvasElement }) => {
+  const canvas = within(canvasElement)
+  const inputField = canvas.getByRole("textbox") as HTMLInputElement
```

Comment:
We don't need to type assert here, right?

```suggestion
  const inputField = canvas.getByRole("textbox")
```